spec/factories/whitehall_import_factory.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :whitehall_import do
+    state { "importing" }
+    payload { build(:whitehall_export_document) }
+    whitehall_document_id { payload["id"] }
+    content_id { payload["content_id"] }
+    document { create(:document, imported_from: "whitehall") }
```

Comment:
can we do this through a regular association? e.g `association :document, imported_from: "whitehall"` ?

Reply:
We sure can, done!